docs/source/authoring_flows.rst

```diff
@@ -427,48 +427,86 @@ An example structure for an ``ExpressionEval`` state is as follows:
 
 All of the properties of the ``ExpressionEval`` state have the same meaning as described in the ``Action`` state. The ``ExpressionEval`` state cannot use the ``InputPath`` property (``Pass`` is appropriate if moving state from an ``InputPath`` to a ``ResultPath`` is needed), so ``Parameters`` must always be present. Just like in ``Action`` the ``Parameters`` may have constant, reference or expression types and portions of the state can be protected using a ``__Private_Parameters`` list. Like ``Action``, this state must have either a ``Next`` or an ``End: true``.
 
+.. _example-flows-custom-format:
+
 ``Globus Web App Custom Formats``
 ---------------------------------
 
-The `Globus web app`_ supports two JSON schema formats in order to make starting flows a little more user friendly on the webapp.
-
-``globus-collection-id``
-^^^^^^^^^^^^^^^^^^^^^^^^
-
-.. code-block:: JSON
-
-    {
-      "source_endpoint_id": {
-        "description": "An example of globus-collection-id.",
-        "format": "globus-collection-id",
-        "title": "Find source collection ID.",
-        "type": "string"
-      }
-    }
+The `Globus web app`_ supports a JSON schema format in order to make starting flows a little more user friendly on the webapp.
 
-``globus-collection-id`` as a ``format`` in your ``input_schema`` will signal to the webapp to show a custom input field for searching for and selecting a Globus collection on the Guided tab when starting a Flow. Note: ``"type": "string"`` should be used in conjuction with this format.
+``globus-collection``
+^^^^^^^^^^^^^^^^^^^^^
 
-.. image:: _static/images/globus-collection-id-ex1.png
-  :alt: Example of the input created by globus collection id format
+``globus-collection`` as a ``format`` in your ``input_schema`` will signal to the webapp to show a custom input field for searching for and selecting a Globus collection on the Guided tab when starting a Flow.
 
-``globus-collection-path``
-^^^^^^^^^^^^^^^^^^^^^^^^^^
+This example input schema shows how to configure a basic Transfer flow using this format:
 
 .. code-block:: JSON
 
```

Comment:
There may be value in adding line emphasis. Sphinx supports a code-block option called [emphasize-lines](https://www.sphinx-doc.org/en/1.4.9/markup/code.html#line-numbers) that might be helpful here:

```rst
.. code-block:: json
   :emphasize-lines: 6, 9-10, 13-19

   {
     "additionalProperties": false,
     ...
```

This example would highlight the `format` line, the `required` lines, and the "id" and "path" property definition lines.